bin/PerlDDS/Run_Test.pm

```diff
@@ -464,7 +464,132 @@ sub wait_kill {
 
   my $process = $self->{processes}->{process}->{$name}->{process};
 
-  return PerlDDS::wait_kill($process, $wait_time, $name, $verbose, $opts);
+  my $result = PerlDDS::wait_kill($process, $wait_time, $name, $verbose, $opts);
+
+  # If opts has indicated that user wants to get core dump/signal information
+  # then find the core dump file if any and print its stack trace.
+  if (defined $opts && defined $opts->{dump_ref} && ${$opts->{dump_ref}}) {
```

Comment:
So is this something we'd have to enable in each test? Is there a way to enable it globally (for example, with an environment variable for everything running on GitHub Actions?)

Reply:
We can make it independent of what users set in `$opts`. I think we always want to print out the stack trace when a core dump is generated. Is there a case where we want to turn it off?